src/main/java/org/folio/service/holdings/AbstractLoadServiceFacade.java

```diff
@@ -115,6 +115,30 @@ private CompletableFuture<HoldingsStatus> populateHoldingsIfNecessary(LoadServic
     });
   }
 
+  private CompletableFuture<HoldingsStatus> waitForStatus(int retryCount, LoadService loadingService) {
+    var future = new CompletableFuture<HoldingsStatus>();
+    waitForStatus(retryCount, future, loadingService);
+    return future;
+  }
+
+  private void waitForStatus(int retries, CompletableFuture<HoldingsStatus> future, LoadService loadingService) {
```

Comment:
I don't think so, because `waitForStatus` method is intended to get first status in order to run `populateHoldingsIfNecessary` method logic and define whether we should wait for `Complete` status or run holdings population.